README.md

```diff
@@ -206,6 +212,46 @@ driver.remove('my_doc_key', (err, res) => {
   if (err) return console.log(err);
 });
 ```
+<a name="Driver+insert"></a>
+
+#### driver.insert(key, value, options, fn)
+Our implementation of <code>Bucket.insert</code> that can recover from temporary errors.
+
+**Kind**: instance method of [<code>Driver</code>](#Driver)  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| key | <code>String</code> | document key to insert |
+| value | <code>String</code> | document contents to insert |
+| options | <code>Object</code> | Options to pass to <code>Bucket.insert</code> |
+| fn | <code>function</code> | callback |
+
+**Example**  
+```js
+driver.insert('my_doc_key', "doc_contents", (err, res) => {
+  if (err) return console.log(err);
+});
+```
+<a name="Driver+upsert"></a>
+
+#### driver.upsert(key, value, options, fn)
+Our implementation of <code>Bucket.upsert</code> that can recover from temporary errors.
+
+**Kind**: instance method of [<code>Driver</code>](#Driver)  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| key | <code>String</code> | document key to upsert |
+| value | <code>String</code> | document contents to upsert |
+| options | <code>Object</code> | Options to pass to <code>Bucket.upsert</code> |
+| fn | <code>function</code> | callback |
+
+**Example**  
+```js
+driver.upsert('my_doc_key', "doc_contents", (err, res) => {
+  if (err) return console.log(err);
+});
+```
```

Comment:
I did generate with 'run docs', but only updated the code comments. I'll add some better detail to the template as well